spark/src/main/java/org/apache/iceberg/spark/actions/BaseExpireSnapshotsSparkAction.java

```diff
@@ -181,10 +184,25 @@ public BaseExpireSnapshotsSparkAction deleteWith(Consumer<String> newDeleteFunc)
 
   @Override
   public ExpireSnapshots.Result execute() {
-    JobGroupInfo info = newJobGroupInfo("EXPIRE-SNAPSHOTS", "EXPIRE-SNAPSHOTS");
+    JobGroupInfo info = newJobGroupInfo("EXPIRE-SNAPSHOTS", getDescription());
     return withJobGroupInfo(info, this::doExecute);
   }
 
+  private String getDescription() {
+    List<String> msgs = new ArrayList<>();
+    if (expireOlderThanValue != null) {
+      msgs.add("older_than=" + expireOlderThanValue);
+    }
+    if (retainLastValue != null) {
+      msgs.add("retain_last=" + retainLastValue);
+    }
+    if (expiredSnapshotIds != null) {
+      msgs.add("Snapshot_ids =" + StringUtils.join(expiredSnapshotIds, ","));
+    }
+    return String.format("Expiring snapshots(%s) in %s", StringUtils.join(msgs, ","), table.name());
```

Comment:
Have you tested this with a very large number of snapshots? I think this is a great idea I just worry what happens if I expire say, 500 snapshots, is the UI still usable?

I think maybe it would be sufficient to list the first and last snapshot being expired.

Reply:
The Spark UI description seems to be a fixed width box, it shows what it can and shows the entire msg as tooltip
![Screen Shot 2021-04-22 at 8 04 26 PM](https://user-images.githubusercontent.com/5082742/115908205-b81fd480-a41e-11eb-9c04-26b1100c02bc.png)
Would mentioning the snapshot id of the first and last convey that only two snapshots are deleted?
Or can we mention the first snapot and mention that there are "n" more snapshots
![Screen Shot 2021-04-22 at 8 17 55 PM](https://user-images.githubusercontent.com/5082742/115907913-58c1c480-a41e-11eb-84c6-8529cb3c46fb.png)

Reply:
I would have said expiring snapshots from
 X to Y

Reply:
I think mentioning the first snapshot eliminated and how many others are being removed is good to